deps/unwind.mk

```diff
@@ -1,4 +1,6 @@
 ## UNWIND ##
+include $(SRCDIR)/unwind.version
```

Comment:
Won't this double-include `unwind.version` since it's a part of `bb-install` now?